docs/HowTo/Use/Multitenancy/Multitenancy.md

```diff
@@ -0,0 +1,154 @@
+# Using multi-tenancy via multiple private states
+
+Use [multi-tenancy via multiple private states (MPS)](../../../Concepts/Multitenancy.md) to allow multiple tenants to
+use the same GoQuorum node, with each tenant having its own private state(s).
+
+## Configure multi-tenancy via multiple private states
+
+### Prerequisites
+
+- [Tessera](https://docs.tessera.consensys.net) version `21.4.0` or later installed
+- GoQuorum installed
+
+!!! important
+
+    If you want to upgrade existing GoQuorum or Tessera nodes to enable multiple private states and multi-tenancy, see
+    the [migration guide](Migration.md).
+
+### Steps
+
+1. Set `isMPS` to `true` in the `config` item of the [GoQuorum genesis file](../../Configure/GenesisOptions.md).
+
+    !!! example "MPS configuration"
+
+        ```json
+        {
+          "config": {
+            ...
+            "isMPS": true
+          },
+          ...
+        }
+        ```
+
+    !!! note
+
+        There can be a mix of MPS-enabled and non-MPS-enabled nodes in a network.
+
+1. Configure the [JSON-RPC security plugin](../JSON-RPC-API-Security.md#configuration).
+   This requires configuring an authorization server.
+   View [examples of configuring the plugin to work with different OAuth2 authorization servers](https://github.com/ConsenSys/quorum-security-plugin-enterprise/tree/master/examples).
+
+1. Set [`enableMultiplePrivateStates`](https://docs.tessera.consensys.net/en/stable/HowTo/Configure/Multiple-private-state/#multiple-private-states)
+   to `true` in the Tessera configuration file.
+   The default is `false`.
+
+    !!! note
+
+        GoQuorum can't start if `isMPS=true` and `enableMultiplePrivateStates=false`.
```

Comment:
Not sure how to highlight the different locations of config. Something like ...

When GoQuorum is started with `isMPS=true` it will query Tessera to ensure that it is configured with `enableMultiplePrivateStates=false`.

What do you think?

Reply:
Steps 1 and 3 clarify the locations of `isMPS` and `enableMultiplePrivateStates`, but I also reworded this note to mention the locations.